src/System.Private.CoreLib/shared/System/Buffers/StandardFormat.cs

```diff
@@ -146,40 +146,47 @@ private static bool ParseHelper(ReadOnlySpan<char> format, out StandardFormat st
         /// </summary>
         public override string ToString()
         {
-            unsafe
+            Span<char> buffer = stackalloc char[FormatStringLength];
+            int charsWritten = Format(buffer);
+            return new string(buffer.Slice(0, charsWritten));
+        }
+
+        /// <summary>The length buffer needed by <see cref="Format"/>.</summary>
```

Comment:
How should I be reading this comment? Should it be: `The buffer length needed by format`?

Reply:
Yes